UPGRADE-6.3.md

```diff
@@ -359,6 +359,7 @@ is recommended (although this tool may not find database-related deprecations).
 - \TPkgCsv2Sql::CreateLogFileName()
 - \TPkgCsv2Sql::GetLogFile()
 - \TTools::AddStaticPageVariables()
+- \CMSTemplateEngine::GetMainNavigation() - splitted in getActiveModuleLayout and filterMainNavigation
```

Comment:
In this list we only tell developers what is deprecated, instructions on what to do should only be noted in the code base itself (source of truth is in the code, and we should avoid duplication). So please remove the instruction (and sort alphabetically).

Reply:
removed redundant info and sorted alphabetically